pom.xml

```diff
@@ -50,22 +57,31 @@
     </repository>
   </repositories>
 
-  <distributionManagement>
-    <repository>
-      <id>folio-nexus</id>
-      <name>FOLIO Release Repository</name>
-      <url>https://repository.folio.org/repository/maven-releases/</url>
-      <uniqueVersion>false</uniqueVersion>
-      <layout>default</layout>
-    </repository>
-    <snapshotRepository>
-      <id>folio-nexus</id>
-      <name>FOLIO Snapshot Repository</name>
-      <uniqueVersion>true</uniqueVersion>
-      <url>https://repository.folio.org/repository/maven-snapshots/</url>
-      <layout>default</layout>
-    </snapshotRepository>
-  </distributionManagement>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.fasterxml.jackson</groupId>
+        <artifactId>jackson-bom</artifactId>
+        <version>${jackson.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.logging.log4j</groupId>
+        <artifactId>log4j-bom</artifactId>
+        <version>${log4j.version}</version>
```

Comment:
Why does this need to be a separate property if it is only used in one place?

Reply:
I agree that it's inconvenient to use variables instead of values directly at dependency's version place, because IDE can't recognize number of version and therefore version suggestions doesn't work. First I though that it's good point to place all the versions in one place and just use names when needed. But considering mentioned above inability of version's completion and the fact that some of values used only once that's a good noticing. I'll fix this.

Reply:
Yeah, I think if the version is used more than once a variable is a good idea.